zds/settings.py

```diff
@@ -493,7 +493,8 @@
         'default_image': os.path.join(BASE_DIR, "fixtures", "noir_black.png"),
         'import_image_prefix': 'archive',
         'build_pdf_when_published': True,
-        'maximum_slug_size': 150
+        'maximum_slug_size': 150,
+        "word_per_minute": 250  # see https://fr.wikipedia.org/wiki/Mot_par_minute
```

Comment:
as you want. Pour l'instant le plus important c'est surtout gérer c e problème de template.